RecoLocalTracker/SiPixelClusterizer/plugins/alpaka/SiPixelRawToClusterKernel.h

```diff
@@ -168,7 +168,7 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE {
                                    const uint32_t numDigis);
 
       SiPixelDigisSoACollection getDigis() {
-        digis_d->setNModulesDigis(nModules_Clusters_h[0], nDigis);
+        digis_d->setNModules(nModules_Clusters_h[0]);
```

Comment:
Now I'm wondering if we could get rid of this completely, and keep the information about the number of modules only on the device.
And if that would help reduce the amount of host/device synchronisations ?

Not for this PR, of course.